cmake/FindPoly.cmake

```
@@ -143,6 +143,20 @@ if(NOT Poly_FOUND_SYSTEM)
       "${DEPS_BASE}/lib/libpicpolyxx${CMAKE_STATIC_LIBRARY_SUFFIX}")
   endif()
 
+  set(POLY_CC_FLAGS )
+  if(NOT(WASM STREQUAL "OFF"))
+    # The flag -Wall is set as default in cvc5, then these flags bellow make
+    # sure the wasm compilation doesn't abort when compiling LibPoly. At
+    # https://github.com/SRI-CSL/libpoly/blob/master/src/upolynomial/factorization.c#L1269-L1274
```

```suggestion
    # https://github.com/SRI-CSL/libpoly/blob/d2cc42c492d05b90069e20ba813ee32fef086566/src/upolynomial/factorization.c#L1269-L1274
```

It is better to link to a specific commit in case that the code gets updated.